tests/unit/test_github_enterprise.py

```diff
@@ -0,0 +1,134 @@
+from github3.github import GitHubEnterprise
+import github3
+
+from .helper import UnitHelper
+
+example_data = {
+    "login": "octocat",
+    "id": 1,
+    "avatar_url": "https://github.com/images/error/octocat_happy.gif",
+    "gravatar_id": "somehexcode",
+    "url": "https://api.github.com/users/octocat",
+    "html_url": "https://github.com/octocat",
+    "followers_url": "https://api.github.com/users/octocat/followers",
+    "following_url": ("https://api.github.com/users/octocat/following"
+                      "{/other_user}"),
+    "gists_url": "https://api.github.com/users/octocat/gists{/gist_id}",
+    "starred_url": ("https://api.github.com/users/octocat/starred"
+                    "{/owner}{/repo}"),
+    "subscriptions_url": "https://api.github.com/users/octocat/subscriptions",
+    "organizations_url": "https://api.github.com/users/octocat/orgs",
+    "repos_url": "https://api.github.com/users/octocat/repos",
+    "events_url": "https://api.github.com/users/octocat/events{/privacy}",
+    "received_events_url": ("https://api.github.com/users/octocat/"
+                            "received_events"),
+    "type": "User",
+    "site_admin": False,
+    "name": "monalisa octocat",
+    "company": "GitHub",
+    "blog": "https://github.com/blog",
+    "location": "San Francisco",
+    "email": "octocat@github.com",
+    "hireable": False,
+    "bio": "There once was...",
+    "public_repos": 2,
+    "public_gists": 1,
+    "followers": 20,
+    "following": 0,
+    "created_at": "2008-01-14T04:33:35Z",
+    "updated_at": "2008-01-14T04:33:35Z"
+}
+
+base_url = 'https://ghe.example.com/'
+
+
+def _build_url(_, *args, **kwargs):
+    return github3.session.GitHubSession().build_url(
+        base_url=base_url + 'api/v3', *args, **kwargs)
```

Comment:
Yes it does, which is quite surprising now that you remind me of this limitation. 

I just ran tox and it works, as it worked before in the lambda.

I thought it might be because it is only called with keyworded arguments everywhere else, but it is not the case so I fail to see why it does work.